server/enums/applicationRole.ts

```diff
@@ -0,0 +1,8 @@
+enum ApplicationRole {
```

Comment:
This enum previously existed in `roleBasedAccessControl` below 👇 
I've moved it here into the `enum` folder as that is a better / more consistent place for it (rather than it being buried in another file)

I've also added the 2 new roles that we will be creating